cmd/security-labeller/main.go

```diff
@@ -36,7 +37,7 @@ func main() {
 	flag.CommandLine = flag.NewFlagSet(os.Args[0], flag.ExitOnError)
 
 	var namespaces arrayFlags
-	flag.Var(&namespaces, "namespace", "Namespace to scan.")
+	flag.Var(&namespaces, "namespaces", "Namespaces to scan, separated by commas. Leave empty to scan all namespaces.")
```

Comment:
Actually, can we specify namespaces separated by commas? Won't that set the element in the array to `"namespace1,namespace2"`.
My understanding was that the user would specify multiple namespace: `--namespace namespace1 --namespace namespace2` and that not specifying `--namespace` at all would give the `NamespaceAll`

Reply:
This should be a `flag.String` then.

Reply:
We can still use `flag.Var`, just needed to change the `Set()` method.